packages/appstore/src/components/add-app/stories/add-app.stories.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import type { Meta, Story } from '@storybook/react';
+import AddApp from '../index';
+
+type AddAppProps = Parameters<typeof AddApp>[0];
+
+export default {
+    title: 'AddApp',
+    argTypes: {
+        is_dark_mode_on: {
+            description: 'Optional',
+            type: 'boolean',
+            defaultValue: false,
+        },
+        is_mobile: {
+            description: 'Optional',
+            type: 'boolean',
+            defaultValue: false,
+        },
+    },
+} as Meta<AddAppProps>;
+
+const Template: Story<AddAppProps> = args => <AddApp {...args} />;
+
+export const AddAppTemplate = Template.bind({});
+AddAppTemplate.args = {
+    apps: [
```

Comment:
apps don't have to be an array. We can change the name to `app` and make it an object instead of an array.
We will be passing only one app.